up-core-api/uprotocol/core/udiscovery/v3/udiscovery.proto

```diff
@@ -41,331 +41,300 @@ service uDiscovery {
   option (version_minor) = 0;
   option (id) = 1;
 
-  // uDiscovery Node Change Notification that sends the Update message
+
   option (notification_topic) = {
     id: 0x8000,
       name: "NodeChange",
       message: "Notification"
   };
 
-    
+  /************************** Write APIs ************************************/
 
-  // Used by any uProtocol application or service to find service instances
-  // locations. The passed UUri contains valid UEntity, UResource, and UAuthority information
-  // for a query.
-  // What is returned is a list of UUris that match the query. Possible return values in status are:
-  // 1. code.OK: Query lookup was successful
-  // 2. code.NOT_FOUND: No matching UUris were found
-  // 3. code.INVALID_ARGUMENT: The passed UUri is invalid
-  // 4. code.PERMISSION_DENIED: The caller does not have permission to perform the query
-  rpc LookupUri(v1.UUri) returns (LookupUriResponse) {
-    option (method_id) = 1;
-  }
+  /* 
+  UEs shall call RegisterUE API to add themselves to discovery.
+ */
```

Comment:
Please provide more description for these APIs as I do not understand their purpose. Also we are revamping the UUri (please watch up-spec project to see what is changing).  comments in method definitions cannot have spaces between them for documentation generation purposes.